app/views/requests/_subject_id_check.html.erb

```diff
@@ -1,8 +1,18 @@
-<%= govuk_panel(html_attributes: { style: "background: #1d70b8; text-align: left; line-height: 1.5em; padding: 25px;" }) do %>
-   <%= @information_request.subject_photo.to_s %>
-   <br>
-   <%= @information_request.subject_proof_of_address.to_s %>
-<% end %>
+ <table class="govuk-table">
```

Comment:
The indentation is off

Reply:
Also, it would be preferable to use the component - https://govuk-components.netlify.app/components/table/